### DISCUSSION TITLE
SlingerPlayer APK on Android TV?

### DISCUSSION THREAD

**Discussion by as400jockey** [GerryDazoo/Slinger#601] [Q&A]
Slingerplayer on my android phone and tablets is awesome.

I want to choose between Firestick and a Google TV stick for an old plasma TV.  I have SlingerPlayer running on a Firestick and it works pretty well.  A little choppy sometimes, whereas on Android it's very fluid. And there's no UI for the remote on Firestick, but no problem - I can run the remote on my android phone from the web page.

I tried sideloading the APK onto my Hisense TV running Google.  I installed the Android app "Send Files to TV" on both my phone and the TV.  I sent the apk file to the TV.  Installed it.  Opened it.  It hung on the splash screen.  

![image](https://github.com/GerryDazoo/Slinger/assets/117923177/a8a8a49b-89cc-4cf3-92d4-4c2b353f8d6e)

I left it for a few minutes, but no joy.

The Back button did nothing.  The Home button took me back to home on GoogleTV.  Slingerplayer does not appear on the list of apps.  Send Files to TV does show on the list.  So it appears that the SlingerPlayer app didn't install completely.

Should this have worked?  Not sure, actually, since I can't rotate the screen to alternate between full screen and remote mode...  so maybe it won't... 

If SlingerPlayer could run on GoogleTV the way it runs on Firestick, that would be great.

**Comment by lead2crm** [comment]
I was using VLC Player to stream my server on my Google TV.

**Comment by bradut** [comment]
I had the same issue with a tablet, and it took me a while [...] to realize that I had to flip it into **portrait** mode to see the menu, same as we use it on the phone. (Probably it wold have helped if there were a message in the main screen indicating this.)

To solve this issue, I've tried to change screen orientation to portrait by using an app (the tablet is located on a different country and I need to remote into it instead of manipulating it directly): 
- it half-worked, none of the apps I've tried satisfied me, and I didn't want to risk messing up the tablet.
- so I went to plan B, already suggested above: **stream using VLC and switch channels using the app on a different device**.
- oh, yes, as I posted in a different thread, I have also experimented with installing a VM on the tablet. The VM 's default display mode was portrait (even when the tablet was in landscape), so the app worked thus giving me the sense that my tablet and the apps were incompatible and had to use the a VM. That was not the case, the app runs OK - just had to flip the tablet, but flipping is not possible on some Android devices such Android boxes (e.g. MINIX) or Android TVs.
